.github/workflows/release.yml

```
@@ -91,72 +91,190 @@ jobs:
               <div class="container">
-                  <h1>🚀 TiLoKit</h1>
-                  <p>Universal CLI Toolkit for Multi-Framework Project Generation</p>
-                  
-                  <h2>🔧 Quick Install</h2>
-                  <div class="install-cmd">curl -fsSL https://tienld-0801.github.io/tilokit/install.sh | bash</div>
+                  <div class="header">
+                      <h1>🚀 TiLoKit</h1>
+                      <p>Universal CLI Toolkit for Multi-Framework Project Generation</p>
+                      <p><strong>Version:</strong> $VERSION | <strong>Build:</strong> $BUILD_DATE</p>
+                  </div>
                   
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)